bugsnag-plugin-android-ndk/src/main/jni/bugsnag_ndk.c

```diff
@@ -42,6 +42,26 @@ bsg_unwinder bsg_configured_unwind_style() {
   return BSG_CUSTOM_UNWIND;
 }
 
+void bugsnag_add_on_error_env(JNIEnv *env, on_error on_error) {
+  if (bsg_global_env != NULL) {
+    bsg_global_env->on_error = on_error;
+  }
+}
+
+void bugsnag_remove_on_error_env(JNIEnv *env, on_error on_error) {
+  if (bsg_global_env != NULL) {
+    bsg_global_env->on_error = NULL;
+  }
+}
+
+bool bsg_run_on_error_cbs(bsg_environment *env) {
+  on_error cb = env->on_error;
+  if (cb!= NULL) {
+      return cb((intptr_t) &env->next_event);
```

Comment:
I'm not sure what this warning is referring to, so believe it might have been addressed by changing `intptr_t` to `void *`. Lmk if that's not the case